lib/semantics/resolve-names.cc

```diff
@@ -2799,15 +2799,32 @@ bool SubprogramVisitor::BeginMpSubprogram(const parser::Name &name) {
   return true;
 }
 
+// A subprogram declared with SUBROUTINE or function
+// If the subprogram is not in interface block, check if its declared as separate 
+// module procedures
 bool SubprogramVisitor::BeginSubprogram(
     const parser::Name &name, Symbol::Flag subpFlag, bool hasModulePrefix) {
+  bool isSeparateModuleProc = false;
```

Comment:
We prefer braced initialization `{false}` when either form will work, as here.

Reply:
I will follow braced initialization.